guide/ops/locations/_azure-classic.md

```diff
@@ -0,0 +1,235 @@
+---
+section: Azure Compute Classic
+section_type: inline
+section_position: 2.1
+---
+
+### Azure Compute Classic
+ 
+Azure is a cloud computing platform and infrastructure created by Microsoft. Apache Brooklyn includes support for both Azure Classic and Azure ARM, as
+one of the [Apache jclouds](http://jclouds.org) supported clouds `Microsoft Azure Compute`.
+
+The two modes of using Azure are the "classic deployment" model and the newer "resource manager deployment" (ARM)
```

Comment:
maybe use "Azure resource manager" rather than "resource manager deployment" so the acronym makes sense?